.github/actions/image-scan/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Image Vulnerability Scan
+description: Run image vulnerability scans
+inputs:
+  image:
+    description: The docker image to scan
+    required: true
+runs:
+  using: "composite"
```

Comment:
What's the value of making this a composite action instead of having the projects call trivy and anchore scans directly? As a point of comparison, [we call tfsec and checkov directly in ci-infra.yml](https://github.com/navapbc/template-infra/blob/main/.github/workflows/ci-infra.yml#L41-L67). One benefit I could see is that it hides away all the parameters. But the disadvantage is it's an extra layer of abstraction that I'm not 100% sure is needed.

Reply:
I think there can be an argument for both cases: 
 - By having it as a composite, you can call that from any pipeline, and expect the output to be the same, and don't need to update in many places when there are any changes you'd like to do
 - By using it directly, you don't have to bother with using composite actions, which can sometimes cause fits because of weird behavior

I can make a workflow that has it directly used instead of calling the action, and we can let the users decide which is best for their setup

Reply:
Can you say more about "By having it as a composite, you can call that from any pipeline, and expect the output to be the same, and don't need to update in many places when there are any changes you'd like to do"? What are some example use cases for scanning the image outside of CI?

Reply:
Meaning if we had the jobs in multiple places instead of a composite, any changes would have to be done to all of them to have the same output. Admittedly, that's not that hard to update it. 

As for outside of CI, in our pipelines right now, we scan in the CI and CD, but I think moving it to just CI is the better move